app/javascript/scavenger_hunt/data/staticData/clues.js

```diff
@@ -1,21 +1,92 @@
 export const Clues = [
   {
-    question: "There’s a student government at UWindsor that represents all undergraduate students in the Faculty of Science. What’s the name of this organization?",
+    puzzleNo: '1',
+    interactive: false,
+    question: "## There’s a student government at UWindsor that represents all undergraduate students in the Faculty of Science. What’s the name of this organization?",
     hint: "",
-    answer: "1"
+    answers: ["Science Society"]
   },
   {
-    question: "What’s Science Society’s website? www.________.com",
+    puzzleNo: '2',
+    interactive: false,
+    question: "## What’s Science Society’s website? www.________.com",
     hint: "",
-    answer: "2"
+    answers: ["uwinscisoc"]
   },
   {
-    question: "Q.3 SciSoc organizes many annual events. In particular, we celebrate two science-related dates every year. What are those days?",
+    puzzleNo: '3',
+    interactive: false,
+    question: "## SciSoc organizes many annual events. In particular, we celebrate two science-related dates every year. What are those days?",
     hint: "Hint: On one day, we give out free guacaMOLE and on the other, it’s free PIe!",
-    answer: "3"
+    answers: ["3"]
   },
   {
-    question: "Here are three clues to letters of the alphabet: ",
-    answer: "4"
+    puzzleNo: '4',
+    interactive: false,
+    question: `<h2> Here are three clues to letters of the alphabet:</h2>`+
```

Comment:
are we going to clean up the question field here? since we have a component for each question, is it still needed to have it as a field in the js?

Reply:
I am working on puzzle 2,3,4,5

Reply:
ok just to make sure, they are going to be cleaned up and put into components?

Reply:
@prairir yes, if you do not need this file for saving the answers then I think we can remove this file. We can save answers in DB and I can create the GQl query for you to use. How does that sound?

Reply:
no i just meant the question part, like the `question: ...` part

Reply:
Got it, removing it